docs/contributing.rst

```diff
@@ -182,145 +226,115 @@ Now that you have a general sense of how the repo is organized (and how you will
 
 .. image:: images/decorative_line4.png
   :width: 700px
   :height: 8px
   :align: center
   :alt: decorative line
 
-* You also want to set your master branch to track the upstream remote:
-
-.. code-block:: bash
+.. note::
     
-    # fetch the latest changes (but doesn't merge remote changes to your local files)
-    git fetch upstream
-    # track the upstream remote branch 
-    git branch -u upstream/master
+    You won't be able to push changes you make directly to the official handbook repo. Instead, you will have to submit pull requests and the moderators/owners of the repo will review your pull request and decide whether they want to accept and merge your changes with the official handbook repo.
 
-.. image:: images/decorative_line4.png
-  :width: 700px
-  :height: 8px
-  :align: center
-  :alt: decorative line
+    For this reason, it is **highly recommended** that you don't make any changes to the `master` branch of your forked repo. Instead, there should be a 1-to-1 relationship between a feature you want to change and a pull request. In other words, you should create a new branch for every issue you want to change so that you can submit separate pull requests for every thing you want to modify.
 
-3. **Pull any new commits in the official handbook repo** to keep your master branch up to date:
+3. When you start working on a new feature (i.e. modification), **create/checkout a new branch** for that feature:
 
 .. code-block:: bash
 
-    # fetches and merges the remote changes to your local files
-    git pull --ff-only
-    # push the commits to your forked repo
-    git push origin master
-
-* You have to do ``git push origin master`` in order to push any commits that you pulled from the official repo to your forked repo. 
+    # create and switch to new branch called 'new-feature' 
+    git checkout -b [new-feature]
+      e.g., git checkout -b testing
 
 .. image:: images/decorative_line4.png
   :width: 700px
   :height: 8px
   :align: center
   :alt: decorative line
 
-.. note::
-    
-    You won't be able to push changes you make directly to the official handbook repo. Instead, you will have to submit pull requests and the moderators/owners of the repo will review your pull request and decide whether they want to accept and merge your changes with the official handbook repo.
-
-    For this reason, it is **highly recommended** that you don't make any changes to the `master` branch of your forked repo. Instead, there should be a 1-to-1 relationship between a feature you want to change and a pull request. In other words, you should create a new branch for every issue you want to change so that you can submit separate pull requests for every thing you want to modify.
+4. Using your favorite source code text editor, **make changes to the the document you want to modify**. Remember, you should be modifying the .rst files located in :blue:`handbook/docs/content_pages`.
```

Comment:
Except for, e.g., the `contributing.rst` etc, which lives in `handbook/docs`

Reply:
LOL yea, but I guess the thought it that people should only be making changes to something in the content_pages folder?...

Reply:
We could soften the language and say something like "........" Actually, I got nothing. @emcdevitt, do you have any thoughts?

Reply:
@snastase @paulapbrooks I'll just add something noting the exception...